client/src/components/TabbedContent.js

```diff
@@ -81,23 +82,26 @@ export class TabbedContent extends React.Component {
         key,
         label: tab_labels[key],
         is_open: open_tab_key === key,
-        is_disabled: (tab_is_disabled)?tab_is_disabled[key]:false,
+        is_disabled: _.includes(disabled_tabs, key),
```

Comment:
Since having no disabled tabs is the default state, wanted the `disabled_tabs` prop to be totally optional. This tweak, and the default empty array value set below, do that.

Reply:
Logic makes sense
👍